views/layout.erb

```diff
@@ -46,12 +46,20 @@
               <a href="/payloads" class="navbar-item">payloads</a>
               <a href="/exploits" class="navbar-item">exploits</a>
               <a href="/db" class="navbar-item">database</a>
-              <a href="/nmap" class="navbar-item">nmap</a>
-              <a href="/masscan" class="navbar-item">masscan</a>
               <a href="/import" class="navbar-item">import</a>
-              <a href="/spider" class="navbar-item">spider</a>
-              <a href="/recon" class="navbar-item">recon</a>
-              <a href="/vulns" class="navbar-item">vulns</a>
+             
```

Comment:
I checked `git log -u` locally and git is saying there's extra whitespace on lines 50 and 62.